src/strategies/multichain/examples.json

```diff
@@ -0,0 +1,55 @@
+[
+  {
+    "name": "Multichain Voting",
+    "strategy": {
+      "name": "multichain",
+      "params": {
+        "strategies": [
+          {
+            "name": "erc20-balance-of",
+            "network": "1",
+            "params": {
+              "address": "0x579cea1889991f68acc35ff5c3dd0621ff29b0c9",
+              "decimals": 18
+            }
+          },
+          {
+            "name": "erc20-balance-of",
+            "network": "137",
+            "params": {
+              "address": "0xB9638272aD6998708de56BBC0A290a1dE534a578",
+              "decimals": 18
+            }
+          },
+          {
+            "name": "erc20-balance-of",
+            "network": "56",
+            "params": {
+              "address": "0x0e37d70b51ffa2b98b4d34a5712c5291115464e3",
+              "decimals": 18
+            }
+          },
+          {
+            "name": "erc20-balance-of",
+            "network": 137,
+            "params": {
+              "address": "0xfC0fA725E8fB4D87c38EcE56e8852258219C64Ee",
+              "decimals": 18
+            }
+          }
+        ],
+        "graphs": {
+          "56": "https://api.thegraph.com/subgraphs/name/apyvision/block-info",
```

Comment:
Any reason someone would want to change these urls? I think it's better to have that urls hardcoded in the strategy directly rather than using the strategy parameters, it will be easier to setup.

Reply:
main reason its to avoid having to update the strategy if new chains are added

Reply:
Sound good!